device_samples/getting_started/src/send_messages/send_messages_to_central.js

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
```

Comment:
>Licensed [](http://example.com/codeflow?start=3&length=8)

I LOVE, love, love how you organized the `send_messages` sample here.  "How can I send a message to iot central?" feels like a very common question, and it's way better to have a single level-100 sample that shows this together instead of forcing the reader to take part of a DPS sample and part of a sendEvent sample and munge them together.

Related side question: should there be samples under `connections` for connecting to central and for connecting using DPS and using the Edge environment?  I know that DPS is a different library, but it's really just a different way to connect your device.

Reply:
+1 on samples under `connections` for Central, DPS, and Edge.